### DISCUSSION TITLE
how to log in after update (5.3 to 6.0) (Ldap problem)

### DISCUSSION THREAD

**Discussion by Lisra2004** [YetiForceCompany/YetiForceCRM#16941] [How to]
impossible to connect following the update (5.3 -> 6.0).
E_WARNING: ldap_bind(): Unable to bind to server: Invalid credentials in /var/www/html/modules/Users/authmethods/Ldap.php, line 54
Do you know how to solve this problem?
When i try to connect with admin log, vtiger return me on login page
Thank you in advance for your answers

**Comment by mariuszkrzaczkowski** [comment]
Make sure you have the correct LDAP configuration and that you have the correct access data

> E_WARNING: ldap_bind(): Unable to bind to server: **Invalid credentials**

**Comment by Lisra2004** [comment]
I have the correct ldap configuration
I solve Ldap problem when i modify line 54
$bind = ldap_bind($ds, $this->userRecordModel->get('user_name') . $auth['domain'], $password);
become:
$bind = ldap_bind($ds, $auth['domain']. '\\' . $this->userRecordModel->get('user_name'), $password);
I always have connection problem, it always loops to send me back to the connection page
no info in browser
nothing in the log files
normally I think I should have a return ( Debug.php => public static $LOG_LEVELS = false;)
